files.geojson

```diff
@@ -2,36 +2,36 @@
-{ "type": "Feature", "properties": { "url": "https://cdn.proj.org/at_bev_GV_Hoehengrid_V1.tif", "name": "at_bev_GV_Hoehengrid_V1.tif", "area_of_use": "Austria", "type": "VERTICAL_OFFSET_VERTICAL_TO_VERTICAL", "source_crs_code": "EPSG:5778", "source_crs_name": "GHA height", "target_crs_code": "EPSG:9274", "target_crs_name": "EVRF2000 Austria height", "source": "Austria Bundesamt für Eich- und Vermessungswessen", "source_country": "Austria", "source_id": "at_bev", "source_url": "http://www.bev.gv.at/portal/page?_pageid=713,2157075&_dad=portal&_schema=PORTAL", "description": "GHA height (EPSG:5778) to EVRF2000 Austria height (EPSG:9274)", "file_size": 1108072, "sha256sum": "f4623c0d37f6dddf0fbfe6230ebd42b40a8744bec94b240075f4f97b90e5dae0" }, "geometry": { "type": "Polygon", "coordinates": [ [ [ 9.5, 46.35 ], [ 17.1875, 46.35 ], [ 17.1875, 49.05 ], [ 9.5, 49.05 ], [ 9.5, 46.35 ] ] ] } },
-{ "type": "Feature", "properties": { "url": "https://cdn.proj.org/at_bev_GV_Hoehengrid_plus_Geoid_V2.tif", "name": "at_bev_GV_Hoehengrid_plus_Geoid_V2.tif", "area_of_use": "Austria", "type": "VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL", "source_crs_code": "EPSG:4937", "source_crs_name": "ETRS89", "target_crs_code": "EPSG:5778", "target_crs_name": "GHA height", "source": "Austria Bundesamt für Eich- und Vermessungswessen", "source_country": "Austria", "source_id": "at_bev", "source_url": "http://www.bev.gv.at/portal/page?_pageid=713,2157075&_dad=portal&_schema=PORTAL", "description": "ETRS89 (EPSG:4937) to GHA Austria height (EPSG:5778)", "file_size": 770542, "sha256sum": "334fa70a877b0fa66026bb52540e7776669bedce58fb6050794f37a4ae0e7f3e" }, "geometry": { "type": "Polygon", "coordinates": [ [ [ 9.5, 46.35 ], [ 17.1875, 46.35 ], [ 17.1875, 49.05 ], [ 9.5, 49.05 ], [ 9.5, 46.35 ] ] ] } },
-{ "type": "Feature", "properties": { "url": "https://cdn.proj.org/au_ga_README.txt", "name": "au_ga_README.txt", "source": "Geoscience Australia", "source_country": "Australia", "source_id": "au_ga", "source_url": "http://www.ga.gov.au", "file_size": 2213, "sha256sum": "82b411bb563187a5ac2c6ba6c2755a6277b6ab17841d56c91e60c6a163e0fc50" }, "geometry": null },
+{ "type": "Feature", "properties": { "url": "https://cdn.proj.org/ar_ign_README.txt", "name": "ar_ign_README.txt", "source": "Instituto GeogrÃ¡fico Nacional (IGN)", "source_country": "Argentina", "source_id": "ar_ign", "source_url": "https://www.ign.gob.ar", "file_size": 777, "sha256sum": "b3813ddb046480e541252ffcb5d4464b94bc013c321ccac8f06b8263a9a51d7f", "version_added": "1.15" }, "geometry": null },
```

Comment:
There are many changes due to different encodings. Apparently there is something different. Do you know what? It changes also the "sha"!

Reply:
I have done a new regeneration of index